www/src/css/_github-markdown.scss

```diff
@@ -1,8 +1,8 @@
 .markdown-body .anchor {
   float: left;
-  line-height: 1;
   margin-left: -20px;
   padding-right: 4px;
+  line-height: 1;
```

Comment:
This is totally an extra thing I did, but I used stylelint to organize all the styles in [logical order](https://www.npmjs.com/package/stylelint-logical-order). I just find it easier to find properties this way if they‘re in the same order each time